src/coreclr/jit/lsra.h

```diff
@@ -1784,6 +1784,12 @@ class LinearScan : public LinearScanInterface
     void clearSpillCost(regNumber reg, var_types regType);
     void updateSpillCost(regNumber reg, Interval* interval);
 
+    FORCEINLINE void updateRegsFreeBusyState(RefPosition& refPosition,
```

Comment:
Is it really necessary to use FORCEINLINE?

Reply:
This is hot functionality used whenever we assign copyRegs with or without consecutive registers. I didn't want to impact the TP for just extracting out this functionality in a method.